README.md

```diff
@@ -83,6 +83,18 @@ export PASS="1"
 ./openvpn-install.sh
 ```
 
+### Headless User Removal
+
+The removal of an existing user can also be fully automated. Again, the key is to provide the (string) value of the `MENU_OPTION` variable along with the remaining mandatory variables before invoking the script.
+
+The following Bash script removes the existing user `bar` from an OpenVPN configuration
+```bash
+#!/bin/bash
```

Comment:
That shouldn't be here

Reply:
Why? It's the same as the example above.